<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="kranadmin" failures="1" tests="3" name="Test" time="7.513" errors="0" timestamp="2025-08-22T12:21:54 IST">
  <testcase classname="listners.OrangeHRM" name="testLogo" time="0.045"/>
  <testcase classname="listners.OrangeHRM" name="testAppUrl" time="0.016">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at listners.OrangeHRM.testAppUrl(OrangeHRM.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testAppUrl -->
  <testcase classname="listners.OrangeHRM" name="testHomePageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testHomePageTitle -->
</testsuite> <!-- Test -->
